docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,321 @@ redirect_from:
+
+This page contains an overview of your app in addition to important credentials you'll need later, like the `Signing Secret` under the **App Credentials** header. 
+
+![Basic Information page](../assets/basic-information-page.png "Basic Information page")
+
+Look around, add an app icon and description, and then let's start configuring your app 🔩
+
+---
+
+### Tokens and installing apps
+Slack apps use [OAuth to manage access to Slack's APIs](https://api.slack.com/docs/oauth). When an app is installed, you'll receive a token that the app can use to call API methods. 
+
+There are two token types available to a Slack app: user (`xoxp`) and bot (`xoxb`) tokens. User tokens allow you to call API methods on behalf of users after they install or authenticate the app. There may be several user tokens for a single workspace. Bot tokens are associated with bot users, and are only granted once in a workspace where someone installs the app. The bot token your app uses will be the same no matter which user performed the installation. Bot tokens are the token type that _most_ apps use.
+
+For brevity, we're going to use bot tokens for this guide.
+
+Navigate to the **OAuth & Permissions** on the left sidebar and scroll down to the **Bot Token Scopes** section. Click **Add an OAuth Scope**.
+
+For now, we'll just add one scope: [`chat:write`](https://api.slack.com/scopes/chat:write). This grants your app the permission to post messages in channels it's a member of.
+
+Scroll up to the top of the OAuth & Permissions page and click **Install App to Workspace**. You'll be led through Slack's OAuth UI, where you should allow your app to be installed to your development workspace.
+
+Once you authorize the installation, you'll land on the **OAuth & Permissions** page and see a **Bot User OAuth Access Token**.
+
+![OAuth Tokens](../assets/bot-token.png "Bot OAuth Token")
+
+> 💡 Treat your token like a password and [keep it safe](https://api.slack.com/docs/oauth-safety). Your app uses it to post and retrieve information from Slack workspaces.
+
+---
+
+### Setting up your local project
+With the initial configuration handled, it's time to set up a new Bolt project. This is where you'll write the code that handles the logic for your app.
+
+If you don’t already have a project, let’s create a new one. Create an empty directory:
+
+```shell
+mkdir first-bolt-app
+cd first-bolt-app
+```
+
+<!-- TODO - Do we want to explain how to install the `pyenv` runtime management in a <details> section? -->
+Next, we recommend using a [Python virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) to manage your project's dependencies. This is a great way to prevent conflicts with your system's Python packages. Let's create and activate a new virtual environment:
+
+```shell
+python3 -m venv .venv
+source .venv/bin/activate
+```
```

Comment:
Do we want to do a better job of offering Windows compatible commands?

Reply:
My concern is that the documentation would bloat as a result. 

Questions on my mind: 

1. Do we know what percentage of our devs are Windows users? 
2. Have any Windows users expressed frustration about existing documentation in the past?

Reply:
I was thinking about this as I was reading - the reason I'd err against it is that the guide uses linux commands above as well. But not a strong objection by any means.

To @misscoded, I haven't seen any complaints about this in the past, but we can pay close attention to it in case it comes up in the future.

Reply:
We don’t need to be creative here. Let’s have a link to venv document. It has a guide for Windows OS users too. https://docs.python.org/3/library/venv.html

Reply:
> 1. Do we know what percentage of our devs are Windows users?

I think we can get a rough number for this, even specifically to Python, from Slack's internal data warehouse. Let's chat about it in our team channel.

My guesstimate is around 50% of Python usage is Windows. Anyone wanna take any bets? 😛 

> 2. Have any Windows users expressed frustration about existing documentation in the past?

Anecdotally, not much with our existing documentation. But I have seen some confusion regarding assumptions we made in comments when describing import paths. Windows has some unique behaviors there.

I believe it is possible to use commands that are identical on Windows for most things. I'd like to see us try to do this, especially because it is more inclusive of developers less like ourselves. If it does look bloaty and there's not a great way to unify, I think we can re-evaluate at the tradeoffs with those specific issues in mind.

Reply:
Thanks for the discussion, everyone! It sounds like the consensus is to use Linux/macOS commands in the guide while linking out to documentation for other operating systems.

The sentence above this code snippet links to the [official Python tutorial on setting up a virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment). It includes commands for both macOS and Windows. It also links to the [official `venv` docs](https://docs.python.org/3/library/venv.html) suggested by @seratch.

I think this gives most developers what they need to setup their environment without bloating our guide. 🙂